src/screens/choice-exercises/services/ChoiceExercisesInterface.ts

```diff
@@ -1,6 +0,0 @@
-import { DocumentsType } from '../../../constants/endpoints'
-import { SingleChoiceListItemType } from '../components/SingleChoiceListItem'
-
-export interface ChoiceExercisesInterface {
```

Comment:
Please restore this and the services. I added them last week to avoid the duplicated code.